myapp/spec/factories/task.rb

```diff
@@ -2,8 +2,8 @@
   factory :task, class: 'Task' do
     title    { 'title' }
     content  { 'content' }
-    user_id  { 1 }
     label    { 'label' }
     status   { 'not_started' }
+    user_id  { FactoryBot.create(:user).id }
```

Comment:
たしか、以下のように書くこともできたと思います。
```suggestion
    user
```
今の状態だとtaskを呼び出すたびにユーザーを作成してしまいますが、
上記のように記載することで、taskオブジェクト作成時の引数にuserを指定すればそのユーザーと紐付き、
引数で指定しなければ新しいユーザーが作成される、といった動きをさせることができたと思いますので、試してみてください。

Reply:
なるほど、ありがとうございます。
上記のように修正しました。